iam/terraform/account-specific/main/circle-ci.tf

```diff
@@ -326,3 +308,40 @@ resource "aws_iam_policy" "circle_ci_policy" {
 
 EOF
 }
+
+resource "aws_iam_user_policy_attachment" "circle_ci_route52_policy_attachment" {
+  user       = aws_iam_user.circle_ci.name
+  policy_arn = aws_iam_policy.circle_ci_route52_policy.arn
+}
+
+resource "aws_iam_policy" "circle_ci_route52_policy" {
+  name = "circle_ci_route52_policy"
```

Comment:
hmm, perhaps a typo? Should this be `route53`?

Reply:
OK will fix it ASAP